CIP-0005/README.md

```diff
@@ -109,15 +109,18 @@ We define the following set of common prefixes with their corresponding semantic
 
 ### Miscellaneous
```

Comment:
Since the DRep and CC encoding are being split we should probably move their definitions to [Hashes subsection](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0005/README.md#hashes).

Reply:
fixed via: [123f487](https://github.com/cardano-foundation/CIPs/pull/737/commits/123f48782ca0181e06b966d6c61b3c08220fc694)